tests/delegation_sandbox.py

```diff
@@ -81,14 +81,14 @@ def setup(controller, key):
     file.write(delegateCode)
 
 try:
-    state = tester.state()
+    state = tester.Chain()
     state.block.number += 2000000
 
-    controller = state.abi_contract(controllerCode)
-    library = state.abi_contract(libraryCode)
+    controller = state.contract(controllerCode, language="serpent")
```

Comment:
Hmm, does `language` no longer default to `serpent`?  I'm not against making this explicit, just curious.